examples/platform/linux/BUILD.gn

```diff
@@ -19,6 +19,10 @@ import("${chip_root}/src/lib/core/core.gni")
 import("${chip_root}/src/lib/lib.gni")
 import("${chip_root}/src/tracing/tracing_args.gni")
 
+declare_args() {
+  chip_enable_smoke_co_trigger = false
+}
```

Comment:
Can we make this to true by default, at least on linux platform ? that way we don't need to add additional build flag and compile the app separately just to test this feature ?

Reply:
@cjandhyala It's set true for all-clusters-app specifically, yes?  Is that not working correctly?

Reply:
It is already true by default on linux platform, see `examples/all-clusters-app/linux/args.gni`.